.github/workflows/python_tests.yml

```
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ "3.8", "3.10" ]
+        python-version: [ "3.8", "3.10", "3.11" ]
```

> So, I don't see a particular reason to fiddle with the version that runs codecov if it ain't broken.

The reason to fiddle with codecov, is that this should just be:

```suggestion
        python-version: [ "3.8", "3.11" ]
```

to bookend our python support, since running with 3.10 is excessive. So if we aren't include 3.10 in the matrix, then we need to change the "if" statements for code cov since currently they are used for 3.10. It can either be used for 3.8 or 3.11 (doesn't really matter).

---

What's the thinking behind needing tests to run on 3.10, if we already run tests for 3.8 and 3.11?